app_dart/lib/src/service/github_service.dart

```diff
@@ -72,53 +74,136 @@ class GithubService {
     }).toList();
   }
 
-  Future<List<PullRequest>> listPullRequests(RepositorySlug slug, String branch) async {
+  /// List pull requests in the repository.
+  Future<List<PullRequest>> listPullRequests(RepositorySlug slug, String branch) {
     ArgumentError.checkNotNull(slug);
-    final PaginationHelper paginationHelper = PaginationHelper(github);
+    return github.pullRequests
```

Comment:
I took a look at the original listPullRequests code, and looks like it does the same thing of what github.pullRequests.list is doing underneath. So I swap it out